.dotnet/src/Custom/Chat/OpenAIChatModelFactory.cs

```diff
@@ -30,37 +30,37 @@ public static ChatCompletion ChatCompletion(
         refusalTokenLogProbabilities ??= new List<ChatTokenLogProbabilityDetails>();
 
         InternalChatCompletionResponseMessage message = new InternalChatCompletionResponseMessage(
-            content,
             refusal,
             toolCalls.ToList(),
             role,
+            content,
             functionCall,
-            serializedAdditionalRawData: null);
+            additionalBinaryDataProperties: null);
 
         InternalCreateChatCompletionResponseChoiceLogprobs logprobs = new InternalCreateChatCompletionResponseChoiceLogprobs(
             contentTokenLogProbabilities.ToList(),
             refusalTokenLogProbabilities.ToList(),
-            serializedAdditionalRawData: null);
+            additionalBinaryDataProperties: null);
 
         IReadOnlyList<InternalCreateChatCompletionResponseChoice> choices = [
             new InternalCreateChatCompletionResponseChoice(
                 finishReason,
                 index: 0,
                 message,
                 logprobs,
-                serializedAdditionalRawData: null)
+                additionalBinaryDataProperties: null)
         ];
 
         return new ChatCompletion(
             id,
-            choices,
-            createdAt,
             model,
-            serviceTier: null,
             systemFingerprint,
-            InternalCreateChatCompletionResponseObject.ChatCompletion,
             usage,
-            serializedAdditionalRawData: null);
+            InternalCreateChatCompletionResponseObject.ChatCompletion,
+            serviceTier: null,
+            choices,
+            createdAt,
+            additionalBinaryDataProperties: null);
     }
```

Comment:
Why did the parameter order change for ChatCompletion?

Reply:
Same as https://github.com/joseharriaga/openai-in-typespec/pull/305#discussion_r1868554559